src/util/config/read-config.ts

```diff
@@ -0,0 +1,20 @@
+import path from 'path';
+import { CantParseJSONFile } from '../errors-ts';
+import readJSONFile from '../read-json-file';
+import { Config } from '../../types';
+import getLocalConfigPath from './local-path';
+
+export default async function readConfig(file?: string) {
+  const pkgFilePath = file || getLocalConfigPath(process.cwd());
+  const result = await readJSONFile(pkgFilePath);
+  if (result instanceof CantParseJSONFile) {
```

Comment:
Let's add a newline above this

Reply:
Updated in `read-config.ts` and `read-package.ts`